android-components/components/browser/engine-gecko/src/main/java/mozilla/components/browser/engine/gecko/prompt/GeckoPromptDelegate.kt

```diff
@@ -282,10 +281,23 @@ internal class GeckoPromptDelegate(private val geckoEngineSession: GeckoEngineSe
         return geckoResult
     }
 
+    companion object {
+        // hint type for auto-generated password
+        internal const val GENERATED = 1
```

Comment:
👍 
Is this value originally derived from an `AutoComplete``GENERATED`? If so perhaps we could directly reference it?